packages/components/src/RecurringSelect/RecurringSelect.mdx

```diff
@@ -0,0 +1,64 @@
+---
+name: RecurringSelect
+menu: Components
+route: /components/recuringselect
```

Comment:
```suggestion
route: /patterns/recurringselect
```

Reply:
(Typo on the component name, and I think moving this one up to Patterns better surfaces the complexity it's handling)